tracer/src/Datadog.Trace/Configuration/IntegrationId.cs

```diff
@@ -67,6 +67,7 @@ internal enum IntegrationId
         SystemRandom,
         AwsDynamoDb,
         HardcodedSecret,
+        TrustBoundaryViolation,
         IbmMq,
         Remoting,
```

Comment:
tiny nit: it's almost certainly fine, but inserting IDs in the middle _changes_ the values of integration IDs. Like I say, I don't think it's actually a problem, even in version conflict, but I'm not 100%, so it's safer to always add at the end

Reply:
Agree. Fixed